pkg/kubectl/cmd/BUILD

```diff
@@ -1,3 +1,5 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
No, please don't do this.  

Don't want everything to be able to depend on this package.